src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -768,30 +787,30 @@ class BasicLayout extends React.PureComponent{
         let ed = new Date(this.state.endDateDual);
         let sd = new Date(this.state.startDateDual);
         if(! this.state.dualLayer) {
-            if (ed > sd && this.state.FormReady != true) {
-                this.setState({FormReady: true})
+            if (ed > sd && this.state.FormReady !== true) {
```

Comment:
use a ternary or helper function to return this value and set the state at the same time.